jacodb-taint-configuration/src/main/kotlin/org/jacodb/taint/configuration/TaintAction.kt

```diff
@@ -46,6 +46,13 @@ val actionModule = SerializersModule {
 
 // TODO add marks for aliases (if you pass an object and return it from the function)
 
+/**
+ * Copies all the marks from the [from] object to the [to].
+ * As a result, all marks from the initial object will be copied into the target object.
+ * If some marks were present only in the [to] object, they remain unchanged. <p>
```

Comment:
`<p>`?

Reply:
It is a way to create a line break in the docs: both HTML and idea rendered

![image](https://github.com/UnitTestBot/jacodb/assets/31047452/8852e032-1d6f-4bf2-a35b-04f2a17cbfc2)